aws_topology/stackstate_checks/aws_topology/resources/dynamodb.py

```diff
@@ -56,37 +56,31 @@ def collect_table_description(self, table_name):
 
     def construct_table_description(self, table_name):
         return {
-            "Table": {
-                "TableName": table_name,
-                "TableArn": self.agent.create_arn("AWS::DynamoDB::Table", self.location_info, resource_id=table_name),
-            }
+            "TableName": table_name,
+            "TableArn": self.agent.create_arn("AWS::DynamoDB::Table", self.location_info, resource_id=table_name),
```

Comment:
Are we trying to create the pre-assumed Table ARN since it already failed in the API call from boto? Why we want to get the false Table from our side?

Reply:
This is for graceful degredation. If the describe_table_description call fails because of no permission, we still need the table ARN to be able to create relations. So, if we can't get the ARN from the actual API, we'll construct it instead.

Reply:
@Eelviny But it's ok if we can't create relations at least we know we couldn't retrieve an item due to a permission issue. We shouldn't ignore the actual problem may be.